Small JS error on pipeline with no jobs in Chrome

Currently using the latest via
`docker run -d -p 8080:8080 -v $PWD:/data gaiapipeline/gaia:latest`

After generating a pipeline (no jobs yet run), i went to the dashboard and got the following error when trying to `start pipeline`.

```
index.vue?258e:110 Uncaught TypeError: Cannot read property 'length' of undefined
    at s.checkPipelineArgs (index.vue?258e:110)
    at s.n [as checkPipelineArgs] (vue.runtime.esm.js:190)
    at click (index.vue?7896:65)
    at t (vue.runtime.esm.js:2000)
    at HTMLAnchorElement.e._withTask.e._withTask (vue.runtime.esm.js:1798)
```

```
    checkPipelineArgs () {
      // check if this pipeline has args
      for (let x = 0, y = this.pipeline.jobs.length; x < y; x++) { <--
        if (this.pipeline.jobs[x].args && this.pipeline.jobs[x].args.type !== 'vault') {
          // we found args. Redirect user to params view.
          this.$router.push({path: '/pipeline/params', query: { pipelineid: this.pipeline.id }})
          return
        }
      }

      // No args. Just start pipeline.
      this.startPipeline()
    },
```
`this.pipeline.jobs` - `jobs` is currently null which is causing the issue

welp, if i read the error message.... this would of made more sense
```
panic: job 'DoSomethingAwesome' has dependency '' which is not declared
```
Sorry about that, closing

--- a/frontend/client/views/pipeline/detail.vue
+++ b/frontend/client/views/pipeline/detail.vue
@@ -233,6 +233,11 @@ export default {
         jobs = pipeline.jobs
       }
 
+      // check if this pipeline has jobs
+      if (!jobs) {
+        return
+      }
+
       // Check if something has changed
       if (this.nodes) {
         var redraw = false
@@ -360,11 +365,13 @@ export default {
 
     checkPipelineArgs () {
       // check if this pipeline has args
-      for (let x = 0, y = this.pipeline.jobs.length; x < y; x++) {
-        if (this.pipeline.jobs[x].args && this.pipeline.jobs[x].args.type !== 'vault') {
-          // we found args. Redirect user to params view.
-          this.$router.push({path: '/pipeline/params', query: { pipelineid: this.pipeline.id }})
-          return
+      if (this.pipeline.jobs) {
+        for (let x = 0, y = this.pipeline.jobs.length; x < y; x++) {
+          if (this.pipeline.jobs[x].args && this.pipeline.jobs[x].args.type !== 'vault') {
+            // we found args. Redirect user to params view.
+            this.$router.push({path: '/pipeline/params', query: { pipelineid: this.pipeline.id }})
+            return
+          }
         }
       }
 

